addons/mrp/models/stock_move.py

```diff
@@ -128,6 +128,13 @@ def default_get(self, fields_list):
                 defaults['additional'] = True
         return defaults
 
+    def _delay_alert_get_documents(self):
+        res = super(StockMove, self)._delay_alert_get_documents()
+        productions = self.mapped('raw_material_production_id')
+        if productions:
+            res['mrp.production'] = productions
+        return res
```

Comment:
move it below _get_upstream_documents_and_responsibles
shouldn't you look also on x.production_id?
shoudln't you call super on self - the records linked to an mo?